contrib/systemd/crio-shutdown.service

```diff
@@ -2,13 +2,11 @@
 Description=Shutdown CRI-O containers before shutting down the system
 Wants=crio.service
 After=crio.service
-Documentation=man:crio(8)
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/rm -f /var/lib/crio/crio.shutdown
-ExecStop=/usr/bin/env bash -c "/usr/bin/mkdir /var/lib/crio; /usr/bin/touch /var/lib/crio/crio.shutdown"
```

Comment:
we still need this. there are some internal processes that need this file to be created

Reply:
both the ExecStart and ExecStop are needed

Reply:
I could only found https://github.com/cri-o/cri-o/pull/4128/files#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bL269-L273 which also mentioned we shouldn't do cleanup with this `/var/lib/crio/crio.shutdown` so I remove it, too.

Ideally, `crio-shutdown.service` will not be triggered during `crio.service` stop/restart so running containers could be keep; only system shutdown/reboot will trigger the `/usr/local/bin/crictl rm -af` before CRI-O daemon gone away.